Deploy/documentation/Mac-Developer-SemanticBits.md

```diff
@@ -745,6 +746,19 @@
    $ brew upgrade node
    ```
 
+## Install or update jq
+
+1. Install Packer using HomeBrew
```

Comment:
@lhanekam You must have meant `jq`

Reply:
Thanks. The risks of copy and paste. :-) I have fixed in my new branch.